coriolis/tasks/replica_tasks.py

```diff
@@ -221,11 +216,11 @@ def run(self, ctxt, instance, origin, destination, task_info,
             migr_connection_info = base.marshal_migr_conn_info(
                 migr_connection_info)
             schemas.validate_value(
-                migr_connection_info,
-                schemas.CORIOLIS_DISK_SYNC_RESOURCES_CONN_INFO_SCHEMA,
-                # NOTE: we avoid raising so that the cleanup task
-                # can [try] to deal with the temporary resources.
-                raise_on_error=False)
+               migr_connection_info,
+               schemas.CORIOLIS_DISK_SYNC_RESOURCES_CONN_INFO_SCHEMA,
```

Comment:
NOTE: the schema wasn't actually updated in any way, and is tailored for SSH (`username` is required):
https://github.com/cloudbase/coriolis/blob/master/coriolis/schemas/disk_sync_resources_conn_info_schema.json#L28

Reply:
This change is only relevant for backup writers, which get created by the target, not the source. The source connection info remains unchanged for now, as does the ```os_morphing``` connection info.

ie: OS morphing is still SSH/WinRM specific